ocaml/idl/datamodel_host.ml

```diff
@@ -2164,6 +2165,16 @@ let t =
             ~default_value:(Some (VEnum "unknown"))
             "Default as 'unknown', 'yes' if the host is up to date with \
              updates synced from remote CDN, otherwise 'no'"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:(Set update_guidances)
+            "pending_guidances_recommended" ~default_value:(Some (VSet []))
+            "The set of pending recommended guidances after applying updates, \
+             which we suggest that most users should follow, but if not \
```

Comment:
How about
```
The set of pending recommended guidances after applying updates, which the most users should follow to make the updates effective. But if not followed, it will not cause a failure.
```?